simularium_metrics_calculator/bin/example.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from simulariumio import InputFileData
+
+from simularium_metrics_calculator import (
+    METRIC_TYPE,
+    PLOT_TYPE,
+    SCATTER_PLOT_MODE,
+    MetricsManager,
+    PlotInfo,
+)
+
+
+def main() -> None:
+    # get the metrics that are available to plot
+    time_metrics = MetricsManager.available_metrics(METRIC_TYPE.PER_TIME)
+    print(f"\nAvailable per time metrics:\n{time_metrics}")
+    agent_metrics = MetricsManager.available_metrics(METRIC_TYPE.PER_AGENT)
```

Comment:
Question: why not just return all available metrics here? Instead of sending in a type. When I looked at `MetricsManager.available_metrics` it's just going over the same data twice and filtering. Maybe this is a nice option, ie sending in the `METRIC_TYPE`, but since each metric has a type on it, it seems useful to just be able to get all the available metrics in one call

Reply:
we should talk about exactly how the front end will request metrics list once we go over that design tomorrow. I can definitely make metric type optional, and maybe we want to adjust this in other ways too.

Reply:
now it's just `metrics = manager.available_metrics()`